package.json

```diff
@@ -67,12 +67,12 @@
     "eslint-formatter-codeframe": "^7.32.1",
     "eslint-plugin-prettier": "^4.0.0",
     "expect.js": "^0.3.1",
-    "knex": "0.95.13",
+    "knex": "1.0.1",
     "mocha": "^9.1.3",
     "mysql": "^2.18.1",
     "pg": "^8.7.1",
     "prettier": "2.4.1",
-    "sqlite3": "^5.0.2",
+    "@vscode/sqlite3": "^5.0.7",
```

Comment:
sqlite3 seems to be more or less abandoned, and there is a known security vulnerability in it, so we were forced to migrate to microsoft fork.